packages/rocketchat-ui/client/lib/textarea-autogrow.js

```diff
@@ -1,92 +1,76 @@
-(function($) {
+(function($)
```

Comment:
This looks like a heavy modification of the Rocket.Chat base feature => It will affect all text areas.
Can we contribute this? If so, we should. Else, we should provide a `requestAutogrow()`-function.

Reply:
@mrsimpson This third party library used in the RC was not up to date with the original. Also, there is no npm/meteor package available for this library. Thus, we need to find a approach to keep this files in sync.
Secondly, the changes to the library includes to avoid flickering of the box and this function is used for message box text areas at this moment. [Link to the original repository](http://github.com/jaz303/jquery-grab-bag/tree/master/javascripts/jquery.autogrow-textarea.js)

Reply:
This lib was not updated in the latest rc version as well. Therefore afaik no other way than updating the file in accordance with the latest version of the lib.